diagnostic.md

```diff
@@ -54,7 +70,12 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+```CSS
+- .big
+- #contentPane
+- .important.red
+- div p
```

Comment:
`div > p` would be the child/parent relationship. the space is a descendent. There is a difference.